### PR TITLE
Updates ARM template files - Stream Analytics

### PR DESCRIPTION
**Pull request by davidsmatlak**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

- Updates files used in ARM template [quickstart](https://docs.microsoft.com/azure/stream-analytics/quick-create-azure-resource-manager) article.
- Adds quickstart article's author to `docOwner` in _metadata.json_.
- Updates _README.md_ with link to the quickstart article.
- Adds **Deploy to Azure Gov** button because _metadata.json_ doesn't specify environments.

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@mamccrea - check this PR for updates that may be needed to documentation that references this sample.

**Comment by davidsmatlak** [comment]
@bmoore-msft The build finished with status **all checks have passed** but the details show **succeeded with issues**. The error shows a Deploy to Azure Gov button is needed. Please clarify if the button is necessary, or if there's other criteria that determines whether or not to add the button?

**Comment by bmoore-msft** [comment]
If you don't specify which environments are supported, the default is "all of them"... in that case you need the button.  If something isn't supported say in USGov, then you need to add the environments property to the metadata.json file and list the supported ones.  E.g. https://github.com/Azure/azure-quickstart-templates/blob/master/201-cdn-with-ruleseengine-rewriteandredirect/metadata.json

**Comment by davidsmatlak** [comment]
@bmoore-msft Thanks for the clarification and that helped me find the some additional info in the contributor's guide. My understanding is that the schema specifies the supported clouds so I'll look for an example.

**Comment by bmoore-msft** [comment]
@mamccrea - check this PR for updates that may be needed to documentation that references this sample.

**Comment by davidsmatlak** [comment]
Changes the `apiVersion` to the latest stable version `2016-03-01` but it created a build error and references version `2019-06-01`. I didn't find a schema for `2019-06-01` but will revert to that version.

**Comment by bmoore-msft** [comment]
@mamccrea - check this PR for updates that may be needed to documentation that references this sample.

**Comment by davidsmatlak** [comment]
@bmoore-msft To clarify, when the _metadata.json_ doesn't list environments, then add the **Deploy to Azure Gov** button. I don't have a way to test or verify environments other than looking at the existing _metadata.json_. I don't want to add unnecessary or incorrect information by including the button for an unsupported environment. Please let me know if there's a better way to verify environments or if I shouldn't add the button to fix build errors? Thanks.

**Comment by bmoore-msft** [comment]
If you don't have a way to test it, just use the pipeline - the majority of cases should work in FF (or we have bigger problems).  Also, if we're submitting a sample (particularly for docs) the eng team should have clear guidance on FF support.

File: 101-streamanalytics-create/README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@
...
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-streamanalytics-create%2Fazuredeploy.json)
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-streamanalytics-create%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-streamanalytics-create%2Fazuredeploy.json)
```
Comment (JasonWHowell) [inline 101-streamanalytics-create/README.md:14]: The link is not HTTPS? they need to fix that.

**Pull request merged**